src/es/components/molecules/sort/Sort.js

```diff
@@ -90,7 +90,7 @@ export default class Sort extends Shadow() {
       }
 
       :host .m-sort__item:hover {
-        background-color: var(--m-gray-100);
+        background-color: #F5F5F5;
```

Comment:
hat es da keine mdx-variable. im mandanten können wir auch die mdx variablen anziehen. (in der Toolbox ist dies anders, dort dürfen keine mdx variablen aber wir haben generell nie hardgecodede Farben, die sollten immer von einer Variable kommen.)

Reply:
Sollte diese Variable sein
```css
--mdx-base-color-grey-100: rgb(245, 245, 245);
```
Quelle: [color.css](https://github.com/mits-gossau/web-components-toolbox-migros-design-experience/blob/b31ff70e5352547e2b6d33edd845fffdbc57cff5/node_modules/%40migros/mdx-design-tokens/dist/css/01_base/color.css#L233C3-L233C28)